web-server/public/js/manager/prosumers-table-updater.js

```diff
@@ -3,7 +3,7 @@
  ******************************************************************************/
 
 
-let intervalUpdateProsumersTable;
+let interval;
```

Comment:
I don't like having a global variable with such a generic name, it is very easy to add another js script to the HTML file with the same global variable name. Change the name of the variable to something more specific.

Reply:
Whoops! Not used to regular javascript, but the name is too long as it was, maybe tableInterval or updateInterval

Reply:
hmm, tableInterval is better but maybe add a p for prosumer. So pTableInterval